app/features/sidebar/pools-section/pool-item.tsx

```diff
@@ -45,38 +47,40 @@ const StyledPoolItem = styled(StyledItem)<{isSelected: boolean}>`
   }
 `
 
-const PoolItem = ({innerRef, styles, data: pool, state, handlers}) => {
+// Is this duplicated somewhere?
```

Comment:
Shouldn't be duplicated (at least not even close to verbatim): this is the new PoolItem for the Pools section tab which is different both in style and code to the current pool list section (which doesn't use the arborist component)